lib/http-server.js

```diff
@@ -45,7 +45,7 @@ var HTTPServer = exports.HTTPServer = function (options) {
 
   if (options.cors) {
     this.headers['Access-Control-Allow-Origin'] = '*';
-    this.headers['Access-Control-Allow-Headers'] = 'Origin, X-Requested-With, Content-Type, Accept';
+    this.headers['Access-Control-Allow-Headers'] = 'Origin, X-Requested-With, Content-Type, Accept, Range';
```

Comment:
Sensible defaults are a good thing, but this code *overrides* any potential specified `options.headers`, instead of merging them. See:

https://github.com/indexzero/http-server/issues/154#issuecomment-106784449